packages/core-contracts/contracts/proxy/ProxyStorage.sol

```diff
@@ -4,7 +4,6 @@ pragma solidity ^0.8.0;
 contract ProxyStorage {
     struct ProxyNamespace {
         address implementation;
-        bool simulatingUpgrade;
```

Comment:
Removed anti-bricking for now. We can re-introduce when complexity stabilizes. And yes, I really want to try again to get rid of this boolean.